tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -1431,6 +1432,7 @@ qos/test_qos_sai.py::TestQosSai::testQosSaiHeadroomPoolWatermark:
          and asic_type in ['cisco-8000']
          and https://github.com/sonic-net/sonic-mgmt/issues/12292 and hwsku in ['Force10-S6100'] and topo_type in ['t1-64-lag']"
       - "topo_type in ['m0', 'mx']"
+      - "'t2' in topo_name and asic_subtype in ['broadcom-dnx']"
```

Comment:
Looks like the T2 DNX linecards are already getting skipped, do we need the DNX check again?
https://github.com/sonic-net/sonic-mgmt/blob/91ea307b26ff2a4f94db2b114de7cd78360416ff/tests/common/plugins/conditional_mark/tests_mark_conditions.yaml#L1478-L1486

Reply:
@arlakshm, removed the condition from testQosSaiHeadroomPoolWatermark.Please review